Gems/Atom/RPI/Code/Source/RPI.Public/Image/ImageSystem.cpp

```diff
@@ -209,6 +209,68 @@ namespace AZ
             return m_systemImages[static_cast<size_t>(simpleImage)];
         }
 
+        const Data::Instance<AttachmentImage>& ImageSystem::GetSystemAttachmentImage(RHI::Format format)
+        {
+            {
+                AZStd::shared_lock<AZStd::shared_mutex> lock(m_systemAttachmentImagesUpdateMutex);
+
+                auto it = m_systemAttachmentImages.find(format);
+                if (it != m_systemAttachmentImages.end())
+                {
+                    return it->second;
+                }
+            }
+
+            // Take a full lock while the map is updated.
+            AZStd::lock_guard<AZStd::shared_mutex> lock(m_systemAttachmentImagesUpdateMutex);
+
+            // Double check map in case another thread created an attachment image for this format while this
+            // thread waited on the lock.
+            auto it = m_systemAttachmentImages.find(format);
+            if (it != m_systemAttachmentImages.end())
+            {
+                return it->second;
+            }
+            
+            RHI::ImageBindFlags formatBindFlag = RHI::ImageBindFlags::Color;
+            RHI::ImageAspectFlags formatAspectFlag = RHI::ImageAspectFlags::Color;
+
+            switch (format)
+            {
+            case RHI::Format::D16_UNORM:
+            case RHI::Format::D32_FLOAT:
+                formatBindFlag = RHI::ImageBindFlags::Depth;
+                formatAspectFlag = RHI::ImageAspectFlags::Depth;
+                break;
+            case RHI::Format::D16_UNORM_S8_UINT:
+            case RHI::Format::D24_UNORM_S8_UINT:
+            case RHI::Format::D32_FLOAT_S8X24_UINT:
+                formatBindFlag = RHI::ImageBindFlags::DepthStencil;
+                formatAspectFlag = RHI::ImageAspectFlags::DepthStencil;
+                break;
+            }
+
+            RHI::ImageDescriptor imageDescriptor;
+            imageDescriptor.m_size = RHI::Size(1, 1, 1);
+            imageDescriptor.m_format = format;
+            imageDescriptor.m_arraySize = 1;
+            imageDescriptor.m_bindFlags = formatBindFlag | RHI::ImageBindFlags::ShaderReadWrite;
+            imageDescriptor.m_sharedQueueMask = RHI::HardwareQueueClassMask::All;
+
+            RPI::CreateAttachmentImageRequest createImageRequest;
+            createImageRequest.m_imagePool = m_systemAttachmentPool.get();
+            createImageRequest.m_imageDescriptor = imageDescriptor;
+            createImageRequest.m_imageName = "SystemAttachmentImage";
+            createImageRequest.m_isUniqueName = false;
+
+            RHI::ImageViewDescriptor viewDesc = RHI::ImageViewDescriptor::Create(imageDescriptor.m_format, 0, 0);
```

Comment:
Should the view descriptor be passed to the createImageRequest?
```suggestion
            createImageRequest.m_imageViewDescriptor = RHI::ImageViewDescriptor::Create(imageDescriptor.m_format, 0, 0);
```

Reply:
After chatting with @VickyAtAZ, looks like we can just remove. Will have a fix up for this shortly.